test/common/stats/thread_local_store_test.cc

```diff
@@ -921,39 +921,39 @@ TEST_F(StatsThreadLocalStoreTestNoFixture, MemoryWithoutTlsFakeSymbolTable) {
   init(true);
   TestUtil::MemoryTest memory_test;
   TestUtil::forEachSampleStat(
-      1000, [this](absl::string_view name) { store_->counter(std::string(name)); });
```

Comment:
note: the changes in this file reduce the scale of these stat memory checks, in order to avoid timeouts in tsan in CI.